### PR TITLE
Refactored anchor server to use polaris

### PR DESCRIPTION
**Pull request by JakeUrban**
I removed code that was moved to the Polaris repository as well as dependencies that are no longer needed, and updated the documentation to reflect the changes. These changes have been tested against the sep24 demo wallet client's deposit and withdrawal flows.

### CODE REVIEW

**Comment by msfeldstein** [comment]
@debnil or @accordeiro can you take a look to make sure i didn't miss anything?

File: Pipfile [hunk 1]
```diff
@@ -3,6 +3,12 @@ name = "pypi"
 url = "https://pypi.org/simple"
 verify_ssl = true
 
+#[[source]]
+#name = "local"
+#url = "http://localhost:3000/simple"
+#url = "http://docker.for.mac.host.internal:3000/simple"
+#verify_ssl = false
```
Comment (accordeiro) [inline Pipfile]: Do we need to keep these?

File: src/app/settings.py [hunk 2]
```diff
@@ -41,66 +30,48 @@
...
-    "django.contrib.staticfiles",
-]
-
-THIRD_PARTY_APPS = ["rest_framework", "corsheaders"]
 
-CUSTOM_APPS = ["info", "transaction"]
-
-INSTALLED_APPS = DJANGO_APPS + THIRD_PARTY_APPS + CUSTOM_APPS
+INSTALLED_APPS = [
+    'django.contrib.admin',
```
Comment (accordeiro) [inline src/app/settings.py]: Any reason why we should convert those back to single quotes? We're currently using the `python-black` formatter which enforces using double quotes instead.

File: Pipfile (hunk 1 above)
Comment (JakeUrban) [reply Pipfile]: No, this was for development. I'll remove it

File: src/app/settings.py (hunk 2 above)
Comment (JakeUrban) [reply src/app/settings.py]: This was copied from an empty django project, I can switch it back.

Comment (msfeldstein) [reply src/app/settings.py]: can we (in a follow up pr) do something to automatically run the black formatter on commit, or configure editors to use black or something so we don't have to get everyone on board individually?

Comment (msfeldstein) [reply src/app/settings.py]: in javascript apps we use `husky` to run git pre-commit hooks to run prettier to autoformat code, as well as set the prettier config to our config so vscode and some other editors auto-format on save

https://github.com/stellar/sep24-demo-client/blob/master/package.json#L34-L39

**Comment by accordeiro** [comment]
One last detail before I forget: besides keeping the `docker-compose.yml` file updated with all background tasks, we should do the same with our `Procfile` to make sure we're running everything we need on https://stellar-anchor-server.herokuapp.com :)

File: src/app/settings.py (hunk 2 above)
Comment (JakeUrban) [reply src/app/settings.py]: You can add a `pre-commit` script in `.git/hooks`:
```
#!/bin/sh

black src
```

**Comment by JakeUrban** [comment]
@accordeiro nice catch, added another `watcher` process for the `check_trustlines` functionality

**Pull request merged**